exonum/src/sandbox/sandbox_tests_helper.rs

```diff
@@ -207,7 +208,9 @@ pub fn compute_txs_merkle_root(txs: &[Hash]) -> Hash {
     let mut fork = MemoryDB::new().fork();
     let mut hashes = ProofListIndex::new("name", &mut fork);
     hashes.extend(txs.iter().cloned());
-    hashes.merkle_root()
+    hashes.merkle_root();
+
+    root_hash(txs)
```

Comment:
Why tho? I'd thought `ProofListIndex::merkle_root()` should be redefined as a part of this PR; leaving it as is and building a full proof on top just creates confusion, IMO. Now, `merkle_root` can be a confusing name (`list_hash` could be better indeed), but having both `merkle_root` and `list_hash` as public methods is IMO quite confusing.